pkg/vectorize/date_format/date_util.go

```diff
@@ -0,0 +1,32 @@
+package date_format
+
+import "strconv"
+
```

Comment:
Move files in pkg/vectorize to sql/plan/function

Reply:
Move the function there.

If the execution is really vectorized, leave the vector impl here.

Reply:
OK, I understand